tailwind.config.ts

```diff
@@ -21,12 +75,15 @@ const config: Config = {
         disabled: colors.gray['400'], // text-disabled
         light: {
           background: colors.gray['100'],
-          disabled: colors.gray['200']
+          disabled: colors.gray['200'],
         },
         dark: {
-          background: colors.zinc['800'],
+          background: newColors['vulcan'].DEFAULT,
           disabled: colors.zinc['600']
         }
+      },
+      backgroundImage: {
+        'bottom-border-glow': `radial-gradient(circle at center, ${newColors.anakiwa.DEFAULT} 0, blue, transparent 100%)`,
```

Comment:
This is the new input bottom border

<img width="575" alt="image" src="https://github.com/reaviz/reablocks/assets/227909/ef3750e7-b568-45ca-a5a8-29c798050add">